src/app/components/Curation/HierarchicalGrid/index.tsx

```diff
@@ -140,9 +140,13 @@ const HiearchicalGrid = ({
                 <Promo.Body className="promo-paragraph" css={styles.body}>
                   {promo.description}
                 </Promo.Body>
-                <Promo.Timestamp className="promo-timestamp">
-                  {promo.lastPublished}
-                </Promo.Timestamp>
+                <React.Fragment>
```

Comment:
I've figured out how we can use `<>` instead of `<React.Fragment>`

At the top of the file, we need to add the following:

```
/* @jsxFrag React.Fragment */
```

This is in the migration of legacy component docs (you need to scroll down to the (index.tsx) section) - https://latest--5d28eb3fe163f6002046d6fa.chromatic.com/?path=/story/coding-standards-migrating-legacy-components--page#after-4